instagram-page-react/src/pages/profilePage/molecules/profileTabbar/ProfileTabbar.js

```diff
@@ -0,0 +1,48 @@
+import React, {useCallback, useState} from 'react';
+
+// Lodash
+import _map from 'lodash/map';
+
+// Styles
+import styles from './profileTabbar.module.css'
+
+// Constants
+import PROFILE_TABBAR_CONFIG, {POSTS_CONFIG} from "./constants/profileTabbar.configs";
+
+// Components
+import ProfileTabbarTab from "./atoms/profileTabbarTab";
+
+function ProfileTabbar() {
+    const [selectedTabId, setSelectedTabId] = useState(POSTS_CONFIG.id);
+
+    const handleProfileTabbarClick = useCallback((tabID) => {
+        setSelectedTabId(tabID);
+    }, [])
```

Comment:
Suggestion: `setSelectedTabId` can be mentioned in dependency array
Although it won't change across re-renders, we can add it to maintain exhaustive dependencies

Reply:
Sure, will do.